test/rewards/AutoCompoundingRewards.ts

```diff
@@ -634,7 +639,50 @@ describe('AutoCompoundingRewards', () => {
                     });
                 });
 
-                describe('processing rewards', () => {
+                describe('the number of programs to auto-process the rewards for', () => {
```

Comment:
`describe` should describe the test group and not the test case itself

Reply:
This IS the test group.
We are doing various checks on the number of programs to auto-process the rewards for:
- Verifying that it is initially larger than zero
- Verifying that it cannot be set to zero
- Verifying that it cannot be set by a non-admin
- Verifying that it can be set by the admin to a value larger than zero

Reply:
BTW, GitHub comparison is a little misleading in this case.
I have not renamed this 'describe' group as this comparison implies.
This is a brand new group added in this PR, right before a 'describe' group which was named 'processing rewards'.
I have renamed that group to 'process rewards', in order for it to be "aligned" with another 'describe' group which I have added in this PR, named 'auto-process rewards'.

Reply:
The name of the brand new `describe` should be this either. My comment is relevant for any `describe` - even if it's a sub-group, etc.

Reply:
done